examples/usecases/entertainment-with-pretrained-embeddings.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<p>I am not sure if we can assuem that workspace for every user. This will likely cause troubles. Can we avoid setting os.chdir?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
good point, removed!

Reply:
yup, removed this